server/graphql/v1/mutations/collectives.js

```diff
@@ -90,6 +94,7 @@ export async function createCollective(_, args, req) {
 
     collectiveData.currency = collectiveData.currency || hostCollective.currency;
     collectiveData.hostFeePercent = hostCollective.hostFeePercent;
+    collectiveData.platformFeePercent = collectiveData.platformFeePercent || hostCollective.platformFeePercent;
```

Comment:
We're now using GraphQL v2 for Collective (type=COLLECTIVE) creation: https://github.com/opencollective/opencollective-api/blob/master/server/graphql/v2/mutation/CreateCollectiveMutations.js